modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -65,6 +67,58 @@ interface ISaveFDSFieldsModalContentProps {
 	saveFDSFieldsURL: string;
 }
 
+interface IContentRendererProps {
+	cetRenderers?: FDSClientExtensionCellRenderer[];
+	item: IFDSField;
+	query: string;
+}
+
+const getRendererLabel = ({
+	cetRenderers = [],
+	rendererName,
+}: {
+	cetRenderers?: FDSClientExtensionCellRenderer[];
+	rendererName: string;
+}): string => {
+	const AVAILABLE_CELL_RENDERERS = [
+		...FDS_INTERNAL_CELL_RENDERERS,
+		...cetRenderers,
+	];
+
+	const renderer = AVAILABLE_CELL_RENDERERS.filter((renderer: any) => {
+		return renderer.name === rendererName || renderer.erc === rendererName;
+	})[0];
```

Comment:
`[0]` will break if `filter` finds no results. Optional chaining or an explicit check should fix this.

Reply:
I would say that it will return an empty array if no result is found, and `undefined` when getting the first position. Something that is fine because we are using the optional chaining for the `renderer?.label` or `renderer?.name`

Reply:
Oh, you are right. I was thinking of `find`, returning `undefined` if an item is not found. `filter` returns an empty array, so this code is fine. Should we use `find`? It does have better performance than `filter`.